internal/model/recommendation.go

```diff
@@ -0,0 +1,18 @@
+package model
+
+import (
+	"github.com/lib/pq"
```

Comment:
Any thoughts about `pgx`? Might be useful in the long run.
Reference: https://preslav.me/2022/05/13/pq-or-pgx-choosing-the-right-postgresql-golang-driver/

Reply:
Gorm internally uses the `pgx`. Here I have used `pq` just to define datatype of containers field on [line 16](https://github.com/RedHatInsights/ros-ocp-backend/pull/11/files#diff-8eb558d7edff2d72730f9cb99e680ea62ad93f2b15bcf2b25a629d51307b5183R16)  Though we can use the [TextArray](https://pkg.go.dev/github.com/jackc/pgtype#TextArray) from the pgtype package but reading the [open issue](https://github.com/jackc/pgx/issues/771) in pgx it is giving sense that using `TextArray` is not convenient. 
Honestly I am not sure with which datatype we should go further or which is easy to implement. I think we can decide this when we will use these structs to marshal/unmarshal data coming through DB.